playbooks/mtls-check.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Ensure mTLS requirements are met
+  hosts: all:localhost
```

Comment:
```suggestion
  hosts: repository:transformers:localhost
```

Running on localhost only was failing, wonder if that would be correct

Reply:
of course the fact needs to be set on all the hosts which use it (not only localhost).

One last thing: it's not really required to run `Assert search-enterprise condition` while doing secrets-init, so I would move the task where it was initially, and then rename this new playbook to facts.yml and we will try to use it to populate facts required by most playbooks

Reply:
Decided to change this line to `hosts: localhost:all:!external` due to errors at: https://github.com/Alfresco/alfresco-ansible-deployment/actions/runs/9740313021/job/26880299719#step:3:1043